lib/thebuggenie/examples/issue_comment.json

```diff
@@ -0,0 +1,19 @@
+{
+  "headers": {},
```

Comment:
Let's add this header as it helps our test script and is probably on the real thing

```
"Content-type": "application/json"
```